strongbox-commons/src/main/resources/application.yaml

```diff
@@ -62,4 +62,31 @@ management:
       exposure:
         include: health,info,beans,metrics,trace,scheduledtasks,threaddump
       base-path: /api/monitoring
-
+cacheManagerConfiguration:
+  groupConfig:
+    name: strongbox
+    password: password
```

Comment:
Could we have this as a non-plain-text value? @strongbox/core-developers : Any suggestions?

And... perhaps the `application.yaml` should only be viewable by the owner (we can control this in the `strongbox-distribution`'s assembly descriptor).

Reply:
> Could we have this as a non-plain-text value? @strongbox/core-developers : Any suggestions?

we can try to research maybe this is possible to use certificate for this

> And... perhaps the application.yaml should only be viewable by the owner (we can control this in the strongbox-distribution's assembly descriptor).

AFAIK this `application.yaml` is under classpath, so you can't directly change or read it

Reply:
What do you think about `jasypt-spring-boot` for encrypting these properties (https://github.com/ulisesbocchio/jasypt-spring-boot)?

The `application.yaml` actually is baked into a jar file (the `strongbox-commons` JAR file which is then included in the spring-boot uber-jar.  Would it be a good convention for the uber-jar to be readable only by the owner?

Reply:
@cbrianhill ,

> The application.yaml actually is baked into a jar file (the strongbox-commons JAR file which is then included in the spring-boot uber-jar. Would it be a good convention for the uber-jar to be readable only by the owner?

We actually need to do something else... 

At the moment, all the properties are being loaded from the classpath. We should check, if they are actually available on the file system and load them from there first and only fallback to the classpath (from within the jar).